SalesforceCommerceCloud/pwa-kit
# Billing address Bug & Automatically select default address

Cannot add a new billing address during checkout: (https://gus.my.salesforce.com/a07AH000000RBv9YAG)
Checkout should automatically select default address if available:(https://gus.my.salesforce.com/a07AH000000U4kRYAS)

How to test-drive this PR
Checkout and run locally
Use a registered customer with saved addresses
confirm that your default address is automatically selected on checkout
confirm that you can add a different billing address and it saves
reference tickets for full repro steps
